assets/js/components/ClusterDetails/SapSystemLink.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
Didn't add an explicit test, as it is tested way better in conjunction with the main views, to have the whole context